tanner/tests/test_mysqli.py

```diff
@@ -4,14 +4,16 @@
 from tanner.utils.asyncmock import AsyncMock
 from tanner.emulators.mysqli import MySQLIEmulator
 
+config = {'host': '127.0.0.1', 'user': 'root', 'password': ''}
+
 
 def mock_config(section, value):
     if section == 'SQLI' and value == 'host':
```

Comment:
These are not needed anymore.

Reply:
Also put the dict inside the mock_config method.

Reply:
Ohh Like that. Okayy